hw_2.py

```diff
@@ -26,7 +116,20 @@
 # encrypt("karaca") ➞ "0c0r0kaca"
 # encrypt("burak") ➞ "k0r3baca"
 # encrypt("alpaca") ➞ "0c0pl0aca"
-
+def encrypt_data(given_input='banana', encrypt_dict=None):
+    """Function implemented processing encrypt given input"""
+    if encrypt_dict is None:
+        encrypt_dict = {'a': '0', 'e': '1', 'i': '2', 'o': '2', 'u': '3'}
+    first_step = [char for char in reversed(given_input)]
```

Comment:
no need to do list comprehension here. You could iterate over reverse(given input)